build.sbt

```diff
@@ -823,7 +823,7 @@ lazy val liteComponentsTestkit = (project in utils("lite-components-testkit")).
   settings(commonSettings).
   settings(
     name := "nussknacker-lite-components-testkit",
-  ).dependsOn(componentsTestkit, requestResponseRuntime, liteEngineRuntime)
+  ).dependsOn(componentsTestkit, requestResponseRuntime, liteEngineRuntime, avroComponentsUtils)
```

Comment:
imo avroComponentsUtils should be dependency of componentsTestkit

Reply:
Can you talk about it with @arkadius ?

Reply:
At the end we want to have kafka avro tests for both lite and flink engine, but at the beginning we could start with test only based on lite engine. If we need it for flink, we will extract it to upper level. WDYT @dswiecki ?